src/app/compiler.js

```diff
@@ -203,7 +203,11 @@ function Compiler (editor, handleGithubCall, outputField, hidingRHP, updateFiles
       for (var fileName in files) {
         var match;
         while ((match = importRegex.exec(files[fileName]))) {
-          importHints.push(match[1]);
```

Comment:
@chriseth I moved the code to under the regex. This solves an immediate problem, so I suggest merging it before we look into how to leverage the compiler to handle the imports (instead of a nasty regex).